protocol-dashboard/src/components/PlaysChart/PlaysChart.tsx

```diff
@@ -8,26 +8,33 @@ type OwnProps = {}
 type PlaysChartProps = OwnProps
 
 const PlaysChart: React.FC<PlaysChartProps> = () => {
-  const [bucket, setBucket] = useState(Bucket.MONTH)
+  const [bucket, setBucket] = useState(Bucket.YEAR)
 
   const { plays } = usePlays(bucket)
-  let error, data, labels
+  let error, data, labels, topNumber: number
   if (plays === MetricError.ERROR) {
     error = true
     labels = []
     data = []
   } else {
     labels = plays?.map(p => p.timestamp) ?? null
     data = plays?.map(p => p.count) ?? null
+    topNumber =
+      plays == null
+        ? null
+        : plays.reduce((acc, p) => {
```

Comment:
nit i like no brackets for simple one-line functions, also we can handle the null gracefully with `plays?.reduce(...)`